src/sidebar/helpers/mentions.ts

```diff
@@ -96,27 +96,53 @@ export function wrapDisplayNameMentions(
 // some assumptions about the tag content in order to be able to get away
 // with it. Specifically we assume that these tags will not contain `<` or
 // `>` in attribute values or the HTML content of the tag.
-const MENTION_TAG_RE = /<a[^>]\bdata-hyp-mention\b[^>]*>([^<]+)<\/a>/g;
+const MENTION_TAG_RE = /<a[^>]\bdata-hyp-mention\b[^>]*>@([^<]+)<\/a>/g;
```

Comment:
I'm updating this to expect the tag content to start with an `@` char. This has some pros and cons:

The main pro is that the logic to get the value without the `@` char is now simpler, as we can rely on the captured value verbatim.

The cons are:

1. We no longer match mention tags if they do not contain the `@` char. I don't know if we'll ever need/want to support this.
2. We don't match mention tags if their content starts with `@` but it's not trimmed (contains empty characters between the opening tag and its content).

I reckon these cons are not really a problem, because the `unwrapMentions` function was already assuming the first character was the `@` character, but there might be other side effects I missed here.